shared/src/business/utilities/getFormattedCaseDetail.ts

```diff
@@ -448,46 +424,16 @@ const formatCounsel = ({ caseDetail, counsel }) => {
   return counsel;
 };
 
-// sort items that do not display a filingDate (based on createdAtFormatted) at the bottom
-export const sortUndefined = (
-  a: { createdAtFormatted: string },
-  b: { createdAtFormatted: string },
-) => {
-  if (a.createdAtFormatted && !b.createdAtFormatted) {
-    return -1;
-  }
-
-  if (!a.createdAtFormatted && b.createdAtFormatted) {
-    return 1;
-  }
-};
-
-export const sortDocketEntries = (
-  docketEntries: (RawDocketEntry & {
-    createdAtFormatted: string | undefined;
-  })[] = [],
-  sortByString = '',
-) => {
-  const sortFunc = getDocketRecordSortFunc(sortByString);
-  const isReversed = sortByString.includes('Desc');
-  docketEntries.sort(sortFunc);
-  if (isReversed) {
-    // reversing AFTER the sort keeps sorting stable
-    return docketEntries.reverse().sort(sortUndefined);
-  }
-  return docketEntries.sort(sortUndefined);
```

Comment:
We were sorting the whole list twice rather than improving our a custom comparison function to allow us to sort once.